samples/net/sockets/civetweb_websocket_server/src/external_mg_cry_internal_impl.inl

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2020 Alexander Kozhinov
+ * Mail: AlexanderKozhinov@yandex.com
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(external_mg_cry_internal_impl, LOG_LEVEL_DBG);
```

Comment:
I would suggest using shorter module name as this makes log lines unnecessary long. Perhaps "mg_int" or similar would be more suitable here.

Reply:
I would prefer `mg_cry_internal_impl`and `LOG_ERR("%s @ %d in civetweb.c", STR_LOG_ALLOC(func), line);` in the function body. Thus the line length printed to the console is reduced and provides better and shorter description of error's root.

Reply:
Ok, no objections if you find the longer string useful.